assets/src/blocks/Articles/ArticlePreview.js

```diff
@@ -0,0 +1,145 @@
+import { Component } from '@wordpress/element';
+const { __ } = wp.i18n;
+
+export class ArticlePreview extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getPageTypesTags(pageType, link) {
+    const { isCampaign } = this.props;
+    const className = `tag-item tag-item--main page-type page-type-${pageType.toLowerCase().replace(' ', '_')}`;
+    if (isCampaign) {
+      return <span className={className}>{pageType}</span>;
+    }
+    return <a className={className} href={link}>{pageType}</a>
+  }
+
+  getAuthorLink() {
+    const {
+      post: {
+        author_name,
+        author,
+        author_override,
+        author_url
+      },
+      isCampaign
+    } = this.props;
+
+    if (author_name) {
+      return (
+        <span className="article-list-item-author">{__('by', 'planet4-blocks')}{' '}
+          {(author_override || isCampaign) ?
+            author_name
+            :
+            <a href={author_url}>{author_name}</a>
+          }
+        </span>
+      )
+    } else if (author) {
+      return (
+        <span className="article-list-item-author">{__('by', 'planet4-blocks')}{' '}
+          {author.is_fake || isCampaign ?
+            author.name
+            :
+            <a href={author.link}>{author.name}</a>
+          }
+        </span>
+      )
+    }
+  }
+
+  render() {
+    const {
+      post: {
+        tags,
+        thumbnail_ratio,
+        thumbnail_url,
+        link,
+        alt_text,
+        page_type,
+        page_type_link,
+        post_title,
+        post_date,
+        post_excerpt
+      },
+      isEditing
+    } = this.props;
+
+    const date = new Date(post_date);
+
+    let articleClassName = "article-list-item";
+    if (tags && tags.length > 0) {
+      tags.forEach(tag => articleClassName += ` ${tag.slug}`);
+    }
+    if (isEditing) articleClassName += ' editing';
```

Comment:
Do we need to set the editing class inside here? Maybe it's easier to set that class on a wrapping element? Maybe here https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/940c184d14d572892b1fa3a9a8400ab54b107cee/assets/src/blocks/Articles/ArticlesEditor.js#L171-L171 ?